src/features/instance/queries/getInstanceInfoQuery.ts

```diff
@@ -37,8 +39,16 @@ type Instance = {
 };
 
 const getInstanceInfo = async (instanceId: string) => {
+	if (isLocalStudio) {
+		return null;
+	}
 	// TODO: Work through any disagreements between the SchemaInstanceType and our local types here.
 	const { data } = await apiClient.get(`/HDBInstance/${instanceId}` as '/HDBInstance/{id}');
+	if (data) {
+		// Set the base URL for the instance client to the first FQDN of the instance
+		// This allows all subsequent API calls to use the correct base URL for the instance
+		instanceClient.defaults.baseURL = data.instanceFqdn;
```

Comment:
When we're doing the "local" version, in a customer's like... deployed instance... how will HDBMS be running? Is it served on the same port as the operations API? And in that case, we're OK not setting the baseURL. If the port is different, we'll find out!